faster_fifo.pyx

```diff
@@ -154,6 +151,21 @@ class Queue:
         return messages
 
     def reallocate_msg_buffer(self, new_size):
-        log.info('Reallocating msg buffer size: %d', new_size)
+        # log.info('Reallocating msg buffer size: %d', new_size)
         self.message_buffer = (ctypes.c_ubyte * new_size)()
         self.message_buffer_memview = memoryview(self.message_buffer)
+
+    def qsize(self):
+        return Q.get_queue_size(<void *>q_addr(self))
+
+    def empty(self):
+        return Q.get_queue_size(<void *>q_addr(self)) == 0
```

Comment:
A minor thing, but why not use `self.qsize` here? You already have this whole line elsewhere why repeat yourself. Although I don't insist here, this is more like a personal preference than the actual suggestion.

Reply:
Perfectly valid. It makes more sense.